### PR TITLE
Pmarash comands realisation

### PR DESCRIPTION
**Pull request by chudopak**
pull request

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (Dmi3Iva) [inline .vscode/settings.json:1]: Please remove this file and add to .ignore .vscode folder

File: srcs/utils.cpp [hunk 2]
```diff
@@ -10,19 +10,34 @@
...
+//   size_t pos = 0;
+//   string token;
+//   while ((pos = s.find(delim)) != std::string::npos) {
+//     token = s.substr(0, pos);
+//     if (!token.empty())
+//       result.push_back(token);
+//     s.erase(0, pos + delim.length());
+//   }
+//   return result;
+// }
```
Comment (Dmi3Iva) [inline srcs/utils.cpp:24]: Please remove commented code

File: srcs/Context/CommandsHeaders.hpp [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#ifndef COMMANDSHEADERS_HPP
```
Comment (Dmi3Iva) [inline srcs/Context/CommandsHeaders.hpp:1]: I understand that you have a lot of includes in vscode, but in Clion we don't have such problem, all includes just shrink to one. Personally, I think that, include for include confuses.

File: srcs/Commands/PrivateMessageCommand.cpp [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+	cout << "CMD " << cmd << endl;
+	cout << "PRIVMSG DO NOTHING FOR NOW" << std::endl;
+
+	if (!user->getIsRegistered()) {
+		cout << "484 * :Your connection is restricted!" << endl;
+		return ;
+	}
+
+	string	receiversStr = cmd.substr(0, cmd.find(' '));
+	// cout << endl << "Receivers " << receiversStr << endl << "Other cmd " << cmd << endl << endl;
```
Comment (Dmi3Iva) [inline srcs/Commands/PrivateMessageCommand.cpp:20]: Please remove comments or uncomment them for debug purpouses.

**Review by Dmi3Iva — COMMENTED** [review]
Please remove some comments, add to ignore vscode file and decide about include in include file

File: srcs/Commands/PrivateMessageCommand.hpp [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+class ACommand;
+
+class PrivateMessageCommand : public ACommand {
+
+public:
+	PrivateMessageCommand(vector<User> *usersPtr, vector<Channel> *channelsPtr);
+	void	execute(User *user, string cmd);
+};
+
+#endif
```
Comment (rxxxt) [inline srcs/Commands/PrivateMessageCommand.hpp:18]: Не хватает новой строки

**Review by rxxxt — APPROVED** [review]

File: srcs/Commands/PrivateMessageCommand.hpp (hunk 5 above)
Comment (Dmi3Iva) [reply srcs/Commands/PrivateMessageCommand.hpp:18]: Я думаю это проблема линтера и его установки, чтобы таких проблем не было нам надо ставить прекоммит, чтобы нельзя было коммитить то что не подходит под линтер
[Линк](https://stackoverflow.com/questions/55965712/how-do-i-add-clang-formatting-to-pre-commit-hook)

File: .vscode/settings.json (hunk 1 above)
Comment (chudopak) [reply .vscode/settings.json:1]: oks

File: srcs/Commands/PrivateMessageCommand.cpp (hunk 4 above)
Comment (chudopak) [reply srcs/Commands/PrivateMessageCommand.cpp:56]: oks

**Pull request closed without merging**